File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task1-Server/BLL/Infrastructure/Models/Institution/InstitutionListItem.cs
using Common.Enums;

namespace BLL.Infrastructure.Models.Institution;
public class InstitutionListItem
{
    public int InstitutionId { get; set; }

    public string Name { get; set; }

    public RatingModel Rating { get; set; }

    public double WeightedRating { get; set; }

    public InstitutionType InstitutionType { get; set; }

    public Region Region { get; set; }

    public string Logo { get; set; }
}

File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task3-WebClient/src/app/core/models/institution/institution-list-item.ts
import { InstitutionType } from "../../enums/institution-type";
import { Region } from "../../enums/region";
import { RatingModel } from "./rating-model";

export interface InstitutionListItem {
	institutionId: number;
	name: string;
	rating: RatingModel;
	weightedRating: number;
	institutionType: InstitutionType;
	region: Region;
	logo: string;
}
